apps/src/lab2/views/Lab2ShareDialogWrapper.tsx

```diff
@@ -51,7 +51,7 @@ const Lab2ShareDialogWrapper: React.FunctionComponent<
     return null;
   }
 
-  if (appType === 'music') {
+  if (appType === 'music' || appType === 'pythonlab') {
```

Comment:
minor/later: would it be worth storing a list of lab2 labs that support the new share dialog in lab2's `constants.ts` instead of needing to update this condition whenever we enable this for a new lab?

Reply:
good call, will update